content/en/network_monitoring/performance/setup.md

```diff
@@ -132,6 +132,9 @@ To enable network performance monitoring with the Datadog Agent, use the followi
 
     **Note**: If the `systemctl` command is not available on your system, run the following command instead: `sudo service datadog-agent restart`
 
+
+6. [Optional] If runing on AWS, enable 'standard collection' for AWS for additional visibility in NAT gateways and Elastic loadbalancers.
```

Comment:
nit: AWS uses "load balancer" as 2 separate words, so I would change this to `Elastic Load Balancers`. Also not sure if we want to mention here ALB/NLB specifically.